bluebrain/repo-patches/packages/meson/package.py

```diff
@@ -5,4 +5,4 @@
 class Meson(BuiltinMeson):
     __doc__ = BuiltinMeson.__doc__
 
-    patch("pgmath.patch", when="target=aarch64:")
```

Comment:
Actually, maybe I was mistaken. Could we revert please :pleading_face:

Reply:
😂 
I think there might be some more targets needed as well - I'll revert, but don't bother approving yet